plugins/catalog-backend/src/ingestion/processors/GithubOrgReaderProcessor.ts

```diff
@@ -112,6 +111,65 @@ export class GithubOrgReaderProcessor implements CatalogProcessor {
 
     return true;
   }
+
+  private async createClient(orgUrl: string): Promise<GraphQL> {
+    let client = this.createClientFromProvider(orgUrl);
+
+    if (!client) {
+      client = await this.createClientFromIntegrations(orgUrl);
+    }
+
+    if (!client) {
+      throw new Error(
+        `There is no GitHub Org provider that matches ${orgUrl}. Please add a configuration for an integration or add an entry for it under catalog.processors.githubOrg.providers.`,
+      );
+    }
+
+    return client;
+  }
+
+  private createClientFromProvider(orgUrl: string): GraphQL | undefined {
+    const provider = this.providers.find(p =>
+      orgUrl.startsWith(`${p.target}/`),
+    );
+
+    if (!provider) {
+      return undefined;
+    }
+
+    this.logger.warn(
+      'GithubOrgReaderProcessor uses provider defined in catalog.processors.githubOrg.providers, migrate to integrations instead.',
```

Comment:
Should (could?) this maybe spit out a link to the docs in some way?  Otherwise, users seeing this warning may have to go digging.